doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -81,6 +81,8 @@ Matter
 * Updated:
 
   * Default heap implementation to use Zephyr's ``sys_heap`` (:kconfig:option:`CONFIG_CHIP_MALLOC_SYS_HEAP`) to better control the RAM usage of Matter applications.
+  * Default output power for FEM devices to +10 dBm.
```

Comment:
Why `+10` and not `+20`? :)

Reply:
Hmm, to be consistent with our tests. For now, we have a +10 dB difference between FEM and non-FEM devices in tests, but it's ok with me to set the maximum value here :)

Reply:
What was the gain prior to this change? I thought it was +20dBm.

Reply:
Previously both FEM, and non-FEM devices were set to +0 dBm, but the FEM Gain was set to +20 dB (wrongly oversteering), and that caused a difference of ~7 dB between those devices. The FEM gain should be set automatically, and the CONFIG_OPENTHREAD_DEFAULT_TX_POWER config sets power at the antenna output only.